### DISCUSSION TITLE
Using an HDMI -> ATSC RF Modulator for HDMI capture is this a crazy idea?

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#124] [General]
I just went from legacy cable to IPTV cable.  This means that my house only needs one coax drop - at my cable gateway/modem.  But my house has cable outlets in several rooms.  I was wondering how to repurpose this coax  throughout my house and decided that I would get an HDMI -> [ATSC RF modulator](https://www.amazon.ca/Modulator-Encoder-Channel-Frequency-Portable/dp/B0CJCQD4L4/?th=1).  For those of us that are old enough this is like when you would plug your VCR or game console into your TV's coax input and tune to channel 3.  So with this method I could distribute video from one HDMI source (more with an HDMI switch) through the house.  You would just need a TV with an ATSC tuner that you would then tune to channel 3 - or whatever channel.

But what about another idea - if you took the HDMI output of your cable box and plugged this into this RF mod and then plugged the coax output into an ATSC tuner like an HDHR?  Shouldn't you be able to use this as a capture device?  I wonder what the video quality will be like?  If you can combine your antenna input and mux in the HDMI channel then you could even use one tuner then you could even record both OTA and Cable with one tuning device - but guide data and channel tuning might be an issue.

![image](https://github.com/OpenSageTV/Community/assets/10366145/5ba34626-639f-4c6c-ab55-b699aabb59a9)

**Comment by wayner9** [comment]
It turns out that the modulator that I received on Amazon only did NTSC, not ATSC. I don't know if that was an error in description or if they sent me the wrong item.

**Comment by emveepee** [comment]
True modulators usually are significantly more so I was skeptical from the beginning (I thought at first you were posting an April  Fool's post).  The title "ATSC or QAM Channel" is deceptive since digital and analog spectrum are the same but searching the one star reviews you would find "This device is intended for use with CCIR cable systems and the channel assignments are set for that system."

With wifi getting faster and faster homes with cable, phone and ethernet jacks are truly becoming obsolete, even if they idea was great at the time.
